main_test.go

```diff
@@ -0,0 +1,40 @@
+package main
+
+import (
+	"io/ioutil"
+	"os"
+	"strings"
+	"testing"
+)
+
+func TestMain__Simple(t *testing.T) {
```

Comment:
One `_` should be enough.

```
func Example() { ... }
func ExampleF() { ... }
func ExampleT() { ... }
func ExampleT_M() { ... }
```
see https://golang.org/pkg/testing/#pkg-overview

Reply:
Thanks for the hint. I was not aware.